saleor/payment/interface.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from decimal import Decimal
+from typing import Dict, Optional
+
+
+@dataclass
+class GatewayResponse:
+    """Dataclass for storing gateway response"""
+
+    is_success: bool
+    kind: str
+    amount: Decimal
+    currency: str
+    transaction_id: str
+    error: Optional[str]
+    raw_response: Optional[Dict[str, str]] = None
+
+
+@dataclass
+class PaymentInformation:
+    """Dataclass for storing all payment information"""
```

Comment:
While this sentence is true, is also a bit vague in my opinion. I guess we should at least say what purpose this dataclass serves. From what I understand, it's used to represent a `Payment` model in a unified manner. Also, I'm wondering about the name as PaymentInformation is quite long. Maybe `PaymentData` would be better? It would somehow fit the "data" naming convention that is used in forms and now in our API as well. I'd like to hear some thoughts on this from others.